### PR TITLE
Fix low hanging fruit in the documentation

### PR DESCRIPTION
**Pull request by gustavstickley**
The docs really need a make over. This was just a cursory effort to clean up the glaring errors.

### CODE REVIEW

**Comment by EverLoSa** [comment]
can you help us to replace all the occurrences of "oni-oa" to "spot-oa",  this is the old name of the project

**Comment by natedogs911** [comment]
Hi @gustavstickley,
I apologize for the delay in merging this,
can you resolve the conflicts so we can merge?

File: doc/index.html [hunk 1]
```diff
@@ -236,16 +236,16 @@
...
 			                <p class="terminal">
 			                    git clone https://github.com/apache/incubator-spot.git
 			                </p><br>
 			
-			                <h3>3.3 Edit apache spot (incubating) configuration.</h3>
+			                <h3>3.3 Edit apache spot (incubating) configuration</h3>
 			                Go to apache spot (incubating) configuration module to edit the solution configuration:<br><br>
 			                <p class="terminal">
-			                    cd /home/solution_user/incubator-spot/spot-setup<br>
+			                    cd /home/solution-user/incubator-spot/spot-setup<br>
```
Comment (rabarona) [inline doc/index.html]: Wondering if there is a way in HTML to represent a placeholder. I think the original reason to use solution_user was that, feels like solution-user is the actual folder, just like incubator-spot. 
The main thing is, solution-user or solution_user is a placeholder for the actual user people choose to use.

File: doc/index.html [hunk 2]
```diff
@@ -452,7 +452,7 @@
 			                    <li>
 			                        Install pip &#45; python package manager.<br>
 			                        <p class="terminal">
-			                            wget --no-check-certificate https://bootstrap.pypa.io/get-pip.py <br>
+			                            wget https://bootstrap.pypa.io/get-pip.py <br>
```
Comment (rabarona) [inline doc/index.html]: What's the reason to use or not use --no-check-certificate?

**Comment by dataminion** [comment]
@gustavstickley can you do a pull on the current branch and resolve your merge conflicts?

**Comment by dataminion** [comment]
@gustavstickley additionally can you please open a ticket in the spot Jira this may feel redundant but the T800 merge bot will not merge unless we can resolve a Jira ticket

https://issues.apache.org/jira/browse/SPOT/

**Comment by kunalnangia** [comment]
Good

**Comment by dataminion** [comment]
this PR is no longer necessary

**Pull request closed without merging**